### PR TITLE
feat(map_tf_generator)!: launching planning_simulator without pointcloud map

### PR DESCRIPTION
**Pull request by shmpwk**
## Description

Adress https://github.com/autowarefoundation/autoware.universe/issues/1203.
Note that this PR will change viewer frame of Rviz.

## Pre-review checklist for the PR author
The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by yukkysaito — APPROVED** [review]
LGTM

File: map/map_tf_generator/Readme.md [hunk 1]
```diff
@@ -2,21 +2,34 @@
...
+To broadcast the `viewer` frame for visualization of map in Rviz.
+The position of the `viewer` frame is the geometric center of input.
 
-Note that there is no module to need `viewer` frames and this is used only for visualization.
+Note that there is no module to need the `viewer` frame and this is used only for visualization.
+
+There are two nodes:
+
+- `pcd_map_tf_generator_node` to visualize pointcloud_map
+- `vector_map_tf_generator_node` to visualize vector_map
```
Comment (kenji-miyake) [inline map/map_tf_generator/Readme.md:13]:
```suggestion
The following are the supported methods to calculate the position of the `viewer` frame:

- `pcd_map_tf_generator_node` outputs the geometric center of all points in the PCD.
- `vector_map_tf_generator_node` outputs the geometric center of all points in the point layer.
```

File: map/map_tf_generator/src/vector_map_tf_generator_node.cpp [hunk 2]
```diff
@@ -0,0 +1,99 @@
+// Copyright 2022 Tier IV, Inc.
```
Comment (kenji-miyake) [inline map/map_tf_generator/src/vector_map_tf_generator_node.cpp:1]:
```suggestion
// Copyright 2022 TIER IV, Inc.
```

**Review by kenji-miyake — COMMENTED** [review]
Generally LGTM. I added some small coments.

File: system/ad_service_state_monitor/config/ad_service_state_monitor.planning_simulation.param.yaml [hunk 3]
```diff
@@ -23,7 +23,6 @@
     topic_configs:
       names: [
         "/map/vector_map",
-        "/map/pointcloud_map",
```
[the remark this answers is not in the document]
Comment (shmpwk) [reply system/ad_service_state_monitor/config/ad_service_state_monitor.planning_simulation.param.yaml:26]: If leaving the pointcloud map topic name, the state won’t transit.

[the remark this answers is not in the document]
Comment (kenji-miyake) [reply system/ad_service_state_monitor/config/ad_service_state_monitor.planning_simulation.param.yaml:26]: Ah, I see. Sorry!

**Comment by shmpwk** [comment]
@kenji-miyake thank you for your checking! I finished fixing.

**Review by kenji-miyake — APPROVED** [review]
Great work! Thank you. :pray:

**Pull request merged**